sdb/commands/count.py

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class Count(sdb.Command):
+    # pylint: disable=too-few-public-methods
+
+    names = ["count", "cnt"]
```

Comment:
`wc`? ;-)

Reply:
"word count" seems like an awkward name, when we're specifically counting "objects" rather than "words".. but I'm not opposed to adding "wc" to the list of names, either.

Reply:
I decided to add it. It seems reasonable if you've been conditioned by `! wc -l` in MDB. Plus it is one less character for the lazy and it can be just typed by your left hand.